src/raise.cr

```diff
@@ -40,17 +40,26 @@ end
 {% if flag?(:win32) %}
   require "callstack/lib_unwind"
 
+  lib LibC
+    fun _CxxThrowException(ex : Void*, throw_info : Void*) : NoReturn
+  end
+
+  @[Primitive(:throw_info)]
+  def throw_info : Void*
+  end
+
   # Raises the *exception*.
   #
   # This will set the exception's callstack if it hasn't been already.
   # Re-raising a previously catched exception won't replace the callstack.
   def raise(exception : Exception) : NoReturn
-    exception.inspect_with_backtrace(STDERR)
-    LibC.exit(1)
-  end
+    {% if flag?(:debug_raise) %}
```

Comment:
I would add a TODO note to remote the whole puts here, or just leave it commented for a while.

Reply:
I don't see why we can't leave the flag in indefinitely.

Reply:
If it's going to stay, then it needs to be supported in the other platforms.

Reply:
Oh yeah, my bad.